### PR TITLE
Update renovate.json

### PR DESCRIPTION
**Pull request by lesv**
reject `-SP.xxx` suffixes in Versions

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -16,6 +16,9 @@
       ],
       "groupName": "AppEngine packages"
     },
+    {
```
Comment (kurtisvg) [inline renovate.json:19]: Do we need to limit to our packages? e.i. "com.google.cloud" or whatever?

File: renovate.json [hunk 2]
```diff
@@ -16,6 +16,9 @@
...
+      "allowedVersions": "!/.+\\-SP\\.[0-9]+$/"
```
Comment (kurtisvg) [inline renovate.json:20]: Should this be:
```suggestion
      "allowedVersions": "!/.+\-SP\.[0-9]+$/"
```

**Review by kurtisvg — CHANGES_REQUESTED** [review]

File: renovate.json (hunk 1 above)
Comment (lesv) [reply renovate.json:19]: I wish, but there is com.google.api, com.google.api-client, as well as com.google.cloud.  (maybe a few more)

**Review by kurtisvg — APPROVED** [review]
LGTM, but still unsure on the regex.

**Review by shubha-rajan — APPROVED** [review]

**Comment by kurtisvg** [comment]
Changed my mind - regex makes sense at second glance.

**Pull request merged**